spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -196,7 +196,7 @@ private void importFileTable(
       importPartitions(table, ImmutableList.of(partition), checkDuplicateFiles);
     } else {
       Preconditions.checkArgument(
-          !partitions.isEmpty(), "Cannot find any matching partitions in table %s", partitions);
```

Comment:
Obviously, `partitions` here is empty and printing is meaningless. We should print the `name of the table`

Reply:
Thanks for fixing this. I think by mistake it has been used `partitions` it should always be 
`table.name()`